src/prices/prices.utils.ts

```diff
@@ -52,6 +56,31 @@ export async function getPrice(address: string, currency?: Currency): Promise<To
   }
 }
 
+export async function fetchPrices(chain: Chain, addresses: string[]): Promise<PriceData> {
```

Comment:
Shall be covered with tests?

Reply:
will be covered in 
https://github.com/Badger-Finance/badger-api/pull/910